### PR TITLE
Fix deprecated URLs

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,11 +38,11 @@ void main() {
 `FakeAsync` can't control the time reported by [`new DateTime.now()`][] or by
 the [`Stopwatch`][] class, since they're not part of `dart:async`. However, if
 you create them using the [`clock`][] package's [`clock.now()`][] or
-[`clock.getStopwatch()`][] functions, `FakeAsync` will automatically override
+[`clock.stopwatch()`][] functions, `FakeAsync` will automatically override
```
Comment (kevmoo) [inline README.md:41]: the `get` version is deprecated

File: pubspec.yaml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 name: fake_async
-version: 1.1.0
-description: Fake asynchronous events such as timers and microtasks for deterministic testing.
-author: Dart Team <misc@dartlang.org>
+version: 1.1.1-dev
+description: >-
+  Fake asynchronous events such as timers and microtasks for deterministic
```
Comment (kevmoo) [inline pubspec.yaml:4]: Move to 2 lines so it's 80 char lines

File: README.md [hunk 3]
```diff
@@ -38,11 +38,11 @@ void main() {
...
 them to use the same notion of time as `dart:async` classes.
 
-[`new DateTime.now()`]: https://api.dartlang.org/stable/dart-core/DateTime/DateTime.now.html
-[`Stopwatch`]: https://api.dartlang.org/stable/dart-core/Stopwatch-class.html
-[`clock`]: https://pub.dartlang.org/packages/clock
-[`clock.now()`]: https://www.dartdocs.org/documentation/clock/latest/clock/Clock/now.html
-[`clock.getStopwatch()`]: https://www.dartdocs.org/documentation/clock/latest/clock/Clock/getStopwatch.html
+[`DateTime.now()`]: https://api.dart.dev/stable/dart-core/DateTime/DateTime.now.html
```
Comment (kwalrath) [inline README.md:44]:
```suggestion
[`new DateTime.now()`]: https://api.dart.dev/stable/dart-core/DateTime/DateTime.now.html
```

**Review by kwalrath — APPROVED** [review]
LGTM once you fix the DateTime.now() link. (It doesn't render as-is.)

File: README.md (hunk 3 above)
Comment (kevmoo) [reply README.md:44]: I'll update the other spot!

**Pull request merged**